src/client/src/orangehrmCoreOAuthPlugin/pages/registerOAuthClient/EditOAuthClient.vue

```diff
@@ -132,8 +161,7 @@ export default {
       isLoading: false,
       oAuthClient: {...initialOAuthClient},
       rules: {
-        clientId: [required, shouldNotExceedCharLength(80)],
-        clientSecret: [required, shouldNotExceedCharLength(80)],
+        name: [required, shouldNotExceedCharLength(80)],
         redirectUri: [shouldNotExceedCharLength(2000)],
```

Comment:
shouldn't we check if valid url? okta only allow valid urls (even http outside localhost is blocked)

![image](https://user-images.githubusercontent.com/12880059/222752466-d84840c7-3f2a-4c21-90b2-34b541361e4e.png)